rex/src/main.rs

```diff
@@ -30,6 +26,13 @@ enum Cmd {
         #[clap(short, long)]
         input: PathBuf,
     },
+    /// Run a benchmark
+    #[command(about = "Run a benchmark")]
+    Benchmark {
```

Comment:
this should not be built into the Rex cli and the benchmarks should just be a cargo test/bench